docs/save_frame_placeholders.md

```diff
@@ -3,9 +3,9 @@
 | Placeholder  | Description                          | Example   |
 | ------------ | ------------------------------------ | --------- |  
 | script_name  | Script name without extension        | haruhi_01 |
-| index        | Video node number (zero-based)       | 0         |
+| index        | Video node number (zero-indexed)     | 0         |
```

Comment:
my brother in christ, lookup before changing stuff, it's zero-based counting system